benchmark_runner/common/oc/oc.py

```diff
@@ -654,6 +654,27 @@ def wait_for_initialized(self, label: str, workload: str = '', status: str = 'In
             else:
                 raise PodNotInitializedTimeout(workload=workload)
 
+    @typechecked
+    @logger_time_stamp
+    def wait_for_csv(self, csv_num: int = 1, namespace: str = environment_variables.environment_variables_dict['namespace'],
+                            timeout: int = int(environment_variables.environment_variables_dict['timeout'])):
+        """
+        This method waits till csv are creating with the required csv number or throw exception after timeout
+        :param csv_num: number of csvs, default 1
+        :param namespace:
+        :param timeout:
+        :return: csv names if getting csv or raise CSVError
+        """
+        current_wait_time = 0
+        while timeout <= 0 or current_wait_time <= timeout:
+            csv_names = self.run(f"{self.__cli} get csv -n {namespace} -o jsonpath={{$.items[*].metadata.name}}")
+            if csv_names and len(csv_names.split()) == csv_num:
+                return csv_names
```

Comment:
Need to handle the case of getting more CSVs than you expect.

Also, do you care what phase they're in at this point?

Reply:
csv_num is a dynamically number according to relevant operator, the default is 1

Reply:
What happens if `oc get csv...` returns _more_ CSVs than what you expect, but includes everything you do want?

Reply:
I can change it to:
`if csv_names and len(csv_names.split()) >= csv_num:`
WDYT ?

Reply:
I think ideally you'd check to make sure that you have all of the specific CSVs that you want, but short of that, I think >= would be a better test.

Reply:
Done